lib/cartopy/feature.py

```diff
@@ -123,6 +123,90 @@ def intersecting_geometries(self, extent):
             return self.geometries()
 
 
+class Scaler(object):
+    """
+    General object for handling the scale of the geometries used in a Feature.
+    """
+    def __init__(self, scale):
+        self._scale = scale
+
+    @property
+    def scale(self):
+        return self._scale
+
+    def scale_from_extent(self, extent):
+        """
+        Given an extent, update the scale.
+
+        Parameters
+        ----------
+        extent
+            The extent over which we should choose a scale. The
```

Comment:
I know what this means, but it would be good to find words to make this even clearer.